roles/bootstrap-os/tasks/bootstrap-oracle.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+- name: Check presence of fastestmirror.conf
+  stat:
+    path: /etc/yum/pluginconf.d/fastestmirror.conf
+  register: fastestmirror
+
+# fastestmirror plugin actually slows down Ansible deployments
+- name: Disable fastestmirror plugin
+  lineinfile:
+    dest: /etc/yum/pluginconf.d/fastestmirror.conf
+    regexp: "^enabled=.*"
+    line: "enabled=0"
+    state: present
+  when: fastestmirror.stat.exists
+
+- name: Get Oracle Linux Version
+  shell: "cat /etc/oracle-release | grep -oE '[0-9]+' | head -1"
+  register: version_result
+
+- name: Set Fact | Oracle Linux Version
+  set_fact:
+    oracle_version: "{{ version_result.stdout | trim}}"
+
+- name: Check EPEL Installed
+  # noqa 303 - checking for existence of package
+  shell: "rpm -qi epel-release"
+  ignore_errors: true
+  register: epel_installed
+
+- name: Get EPEL Packages
+  get_url:
+    url: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{oracle_version}}.noarch.rpm"
+    dest: "/tmp/epel-release-latest-{{oracle_version}}.noarch.rpm"
+  when: epel_installed.rc != 0
+
+- name: Install EPEL Packages
+  yum:
+    name: "/tmp/epel-release-latest-{{oracle_version}}.noarch.rpm"
+    state: present
+  environment: "{{proxy_env}}"
+  when: epel_installed.rc != 0
+
+- name: Check SELinux Installed
+  # noqa 303 - checking for existence of package
+  shell: "rpm -qi container-selinux"
+  ignore_errors: true
+  register: selinux_installed
+
+- name: Get SELinux Packages
+  get_url:
+    url: "http://mirror.centos.org/centos/7/extras/x86_64/Packages/container-selinux-2.68-1.el7.noarch.rpm"
+    dest: "/tmp/container-selinux-{{oracle_version}}.noarch.rpm"
+  when: selinux_installed.rc != 0
+
+- name: Install SELinux Packages
+  yum:
+    name: "/tmp/container-selinux-{{oracle_version}}.noarch.rpm"
+    state: present
+  environment: "{{proxy_env}}"
+  when: selinux_installed.rc != 0
+
+- name: Install packages requirements for bootstrap
+  yum:
+    name: "{{ packages }}"
+    state: present
+  environment: "{{proxy_env}}"
+  vars:
+    packages:
+      - libselinux-python
+
+- name: Install pip for bootstrap
+  yum:
+    name: python-pip
+    state: present
+  environment: "{{proxy_env}}"
+
+- name: Install required python modules
+  pip:
```

Comment:
Do you really want to install packages with `pip` on production machines?

Reply:
A package(s) that solve these dependencies would probably be better in general; but this is the pattern other distros are using.

Reply:
These Python modules (packaged or not) are not needed on any other distro (including CentOS and Fedora) in the first place, why does Oracle Linux need them?